### ISSUE TITLE
[MAS1.1.1] Alt is not descriptive for '3 **** (856)$$$$' text which appears after hit the 'Show review using  Adaptive card' button.

### ISSUE THREAD

**Issue by priysn** [Microsoft/BotFramework-WebChat#1363]
**Actual Result:**
Alt is not descriptive for '3 **** (856)$$$$' text.
When focus lands on the '3 **** (856)$$$$' text then narrator is reading '3(856)$$$$', Not give the information about the starts.

**Expected Result:**
Alt should be more descriptive for for '3 **** (856)$$$$' text.

**Repro Steps:**

1. Open URL [https://microsoft.github.io/BotFramework-WebChat/full-bundle/](https://microsoft.github.io/BotFramework-WebChat/full-bundle/) in Edge browser.
2. Notice 'Upload File' and 'Send' button icons present in bottom of screen.
3. Type 'Help' in the input control at the bottom of the screen and hit enter (or click on send button).
4. Content will upload, Navigate to any "Show review using  Adaptive card" button by using Tab key and press Enter on it.
5. New data will upload.
6. Press F12 and open developer toolbar and check 'Alt' for '3 **** (856)$$$$' text.
7. Observe that Alt is not descriptive for '3 **** (856)$$$$' text.

**User Impact:**
The screen reader will face difficulty if screen reader is not announce name of text in same manner as it is visible on the screen.

**MAS Reference:**
[MAS 1.1.1 - Non-text Content (20)](https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={d2d2051f-bdc8-4af7-8e18-38aae867e216})

**Test Environment:**
**OS:** Windows 10
**OS Build:** 17134.345
**OS Version:** 1803
**Browser:** Edge
**Tool:** Narrator

**Attachment:**
1. ![mas1 1 1_alt is not descriptive for starts](https://user-images.githubusercontent.com/41668935/48547202-0e07fe00-e8f0-11e8-8fe2-aacd0ffe9641.png)
2. [MAS1.1.1_Alt for text.zip](https://github.com/Microsoft/BotFramework-WebChat/files/2584877/MAS1.1.1_Alt.for.text.zip)

**Comment by compulim** [comment]
There seems no way to put "screen reader only" content in a Markdown:
- Although Markdown honor some HTML, I think it don't honor inline style inside the HTML for obvious reason

We could probably leverage `fallbackText` from Adaptive Card JSON, https://docs.microsoft.com/en-us/adaptive-cards/authoring-cards/card-schema#schema-adaptivecard.

# Note to fixer
- [ ] Modify MockBot to include `fallbackText` for this card
- [ ] Add `fallbackText` as `title` attribute on the container of the Adaptive Card

**Comment by corinagum** [comment]
Bot-specific implementation, we will fix MockBot

**Comment by Amit8527510735** [comment]
#HCL;#Accessibility;#HCL_BotFramework_WebChat;A11yMAS;WCAG1.1.1
